README.md

```diff
@@ -1,7 +1,7 @@
-Esendex .Net SDK
+Commify .Net SDK
 ================
 
-The Esendex .Net SDK is an easy to use client for Esendex's REST API which you can use to integrate SMS and Voice messaging into any application built with C#, VB.NET, Managed C++, F# or any other language built on the .NET framework. 
```

Comment:
We may be the Commify group now, but isn't it still the API for the Esendex product?

Reply:
I wondered the exact same thing but it stated in the story that "Readme should reference Commify instead of Esendex." I shall query with the boss

Reply:
Fair enough, but I think it's going to be confusing!

Reply:
Turns out it's just Esendex as a company that needs changing rather than Esendex as a brand so I shall revert these Commify references to Esendex.